README.md

```diff
@@ -32,7 +32,7 @@ textfile | Exposes statistics read from local disk. The `--collector.textfile.di
 time | Exposes the current system time. | _any_
 vmstat | Exposes statistics from `/proc/vmstat`. | Linux
 version | Exposes node\_exporter version. | _any_
-
+zfs | Exposes [ZFS](http://open-zfs.org/) performance statistics.<br/> FreeBSD (ARC, zpool), Linux (ARC) | [FreeBSD](https://www.freebsd.org/doc/handbook/zfs.html), [Linux](http://zfsonlinux.org/)
```

Comment:
The general rule is that if it behaves sanely when zfs isn't loaded, then it's okay to have it on by default. We should aim for as much as is sane to work out of the box.